### ISSUE TITLE
Do you provide codes for joint TTS and ASR?

### ISSUE THREAD

**Issue by xmuspeech** [ShigekiKarita/espnet-semi-supervised#7]

**Comment by ShigekiKarita** [comment]
Yes. I will push it in a few days (but maybe obsolete for the latest
pytorch). Thanks for your request.

2020年2月10日(月) 17:40 xmuspeech <notifications@github.com>:

**Comment by ShigekiKarita** [comment]
Hi, sorry for late. I have uploaded my working branch as-is (not organized yet). https://github.com/ShigekiKarita/espnet-semi-supervised/blob/karita-asrtts/egs/librispeech/asrtts1/note/results.org
